### PR TITLE
cd command

### PR DESCRIPTION
**Pull request by bpinto**
Jump around Oh My Fish packages and themes using `omf cd <name>`. If no
<name> is specified, jump to oh my fish root folder.

### CODE REVIEW

File: pkg/omf/cli/omf.cd.fish [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# SYNOPSIS
+#   Jump around Oh My Fish packages and themes
+#
+# OPTIONS
+#   <name> Name of the package or theme.
+
+function omf.cd -a name -d "Jump around Oh My Fish packages and themes"
```
Comment (bucaran) [inline pkg/omf/cli/omf.cd.fish]: Set PWD to the plugin / theme home directory.

**Comment by bucaran** [comment]
LGTM!

File: pkg/omf/cli/omf.cd.fish [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  if test -z $name
+    pushd $OMF_PATH
+    return 0
+  end
+
+  if test -e $OMF_CONFIG/pkg/$name
+    pushd $OMF_CONFIG/pkg/$name
```
Comment (bucaran) [inline pkg/omf/cli/omf.cd.fish]: Change all instances of `pkg` to `plugin`. I've renamed it everywhere already.

File: pkg/omf/cli/omf.cd.fish (hunk 1 above)
Comment (bpinto) [inline pkg/omf/cli/omf.cd.fish]: :+1: 
Used the description of `cd`

> NAME
       cd -- change directory

  > Synopsis
       cd [DIRECTORY]

>   Description
       cd changes the current working directory.

File: pkg/omf/cli/omf.cd.fish (hunk 2 above)
Comment (bpinto) [inline pkg/omf/cli/omf.cd.fish]: If I rename it's not going to work until you release it.

Comment (bobthecow) [inline pkg/omf/cli/omf.cd.fish]: oh, are we going back to plugin? sweet :)

Comment (bpinto) [inline pkg/omf/cli/omf.cd.fish]: @bucaran just be aware of backwards compatibility... you might need to keep some come around or come up with a `mv $dir/pkg $dir/plugin` on load.

Comment (bobthecow) [inline pkg/omf/cli/omf.cd.fish]: `mv` for the ones omf installed under `.local`, and detect the presence of `$OMF_CONFIG/pkg` and tell the user to move those ones.

Comment (bucaran) [inline pkg/omf/cli/omf.cd.fish]: Right.

**Pull request merged**